test/cctest/test_cppgc.cc

```diff
@@ -23,21 +19,18 @@ class CppGCed : public cppgc::GarbageCollected<CppGCed> {
   static void New(const v8::FunctionCallbackInfo<v8::Value>& args) {
     v8::Isolate* isolate = args.GetIsolate();
     v8::Local<v8::Object> js_object = args.This();
-    CppGCed* gc_object = cppgc::MakeGarbageCollected<CppGCed>(
-        isolate->GetCppHeap()->GetAllocationHandle());
-    js_object->SetAlignedPointerInInternalField(kWrappableTypeIndex,
-                                                &kEmbedderID);
-    js_object->SetAlignedPointerInInternalField(kWrappableInstanceIndex,
```

Comment:
The embedder should still set the instance index slot since there is no pair function for `node::SetCppgcReference` like `node::GetCppgcReference`. Probably not an issue for this test because there are no wrapped object methods.

Reply:
Yeah though I suppose that should be a separate doc/test change…